OpenVBX/views/messages/details.php

```diff
@@ -124,6 +124,43 @@
 			<h3>Transcription</h3>
 			<?php endif; ?>
 			<div class="message-transcript"><?php echo (is_null($message->content_text) ? "(no transcription)" : $message->content_text) ?></div>
+			<?php if(substr($message->call_sid,0,2) == 'MM'):
+				echo "<br><br><h3>Media Attachements</h3>";
+				
+				$ci = & get_instance();
```

Comment:
This is almost all controller logic. Move the logic to `controllers/messages/details.php` and include a 'media' member in the `$details` array. Default it to an empty array.

Then here you can check `if (count($media))` to add the appropriate output.